mods/linux-style-window-drag.wh.cpp

```diff
@@ -0,0 +1,148 @@
+// ==WindhawkMod==
+// @id              linux-style-window-drag
+// @name            Linux-style Window Drag
+// @description     Enables moving windows by clicking anywhere while holding the Win key, similar to GNOME or KDE. Includes a fix for the Start Menu.
+// @version         1.0
+// @author          djestick
+// @github          https://github.com/djestick/linux-style-window-drag-on-windows
+// @include         explorer.exe
```

Comment:
Please see https://github.com/ramensoftware/windhawk/wiki/Mods-as-tools:-Running-mods-in-a-dedicated-process